src/binder/expression_binder.cpp

```diff
@@ -73,35 +73,41 @@ std::shared_ptr<Expression> ExpressionBinder::foldExpression(
     return result;
 }
 
+static std::string unsupportedImplicitCastException(
```

Comment:
We should move this to ExceptionMessage class.

Reply:
I would rather not do so if the exception is used in the same cpp file.